demos/smart_classroom_demo/main.cpp

```diff
@@ -67,13 +89,45 @@ class Visualizer {
 
     void Show() const {
         if (enabled_) {
-            cv::imshow(window_name_, frame_);
+            cv::imshow(main_window_name_, frame_);
         }
+
         if (writer_.isOpened()) {
             writer_ << frame_;
         }
     }
 
+    void DrawCrop(cv::Rect roi, int id, const cv::Scalar& color) const {
+        if (enabled_ && num_top_persons_ > 0) {
```

Comment:
Since the whole body of the fuinction inside this condition, please, negate this condition, and if the negated condition is true, make `return`

Reply:
Done